src/root/views/Search/SurgeAlertTable/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+import { Link } from 'react-router-dom';
+
+import Container from '#components/Container';
+import Table from '#components/Table';
+import LanguageContext from '#root/languageContext';
+import {
+  createDateColumn,
+  createNumberColumn,
+  createStringColumn,
+} from '#components/Table/predefinedColumns';
+
+import { SurgeAlert } from '../index';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  className?: string;
+  data: SurgeAlert[] | undefined;
+}
+
+function surgeAlertKeySelector(surgeAlert: SurgeAlert) {
+  return surgeAlert.id;
+}
+
+function SurgeAlertTable(props: Props) {
+  const {
+    className,
+    data,
+  } = props;
+
+  const { strings } = React.useContext(LanguageContext);
+
+  const columns = [
+    createDateColumn<SurgeAlert, number>(
+      'alert_date',
+      'Alert Date',
+      (surgeAlert) => surgeAlert.alert_date,
+    ),
+    createDateColumn<SurgeAlert, number>(
+      'deadline',
+      'Application Deadline',
+      (surgeAlert) => surgeAlert.deadline,
+    ),
+    createNumberColumn<SurgeAlert, number>(
+      'duration',
+      'Duration',
+      (surgeAlert) => {
+        const duration = Number(surgeAlert.alert_date) - Number(surgeAlert.deadline);
+        return duration;
+      }),
+    createDateColumn<SurgeAlert, number>(
+      'start_date',
+      'Start Date',
+      (surgeAlert) => surgeAlert.start_date
+    ),
+    createStringColumn<SurgeAlert, number>(
+      'name',
+      'Position',
+      (surgeAlert) => surgeAlert.name
```

Comment:
```suggestion
      (surgeAlert) => surgeAlert.name,
```

Reply:
we should follow a project-specific pattern. Please verify how it's done in this particular project and make changes likewise.

Reply:
On the other note, don't we need translations for each of these strings?